### PR TITLE
Get started nav update

### PR DESCRIPTION
**Pull request by fabriguespe**
We observed feedback from devs that the current 'Get Started' section can be a bit overwhelming with all its different topics. How about we streamline it a bit for easier navigation? My suggestion is to break it down into distinct subsections:

- Overview: A snapshot of code examples in various languages.
- Quickstarts: A dedicated space for quickstarts, sorted by language.
- Example apps: Showcasing more detailed sample apps crafted by our Engineering team.
- Reference docs: Tailored for advanced devs, focusing on SDK specifications.

These sections could evolve and grow over time with room to grow into more elaborate and visually appealing sections.

Preview: https://junk-range-possible-git-get-started-navupdate-xmtp-labs.vercel.app/docs/introduction

### CODE REVIEW

File: docs/build/conversations.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 sidebar_label: Conversations
-sidebar_position: 3
+sidebar_position: 4
```
Comment (jhaaaa) [inline docs/build/conversations.md:3]: Love this change - the reordering makes so much sense!

File: docs/build/get-started/_category_.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "label": "Get Started",
```
Comment (jhaaaa) [inline docs/build/get-started/_category_.json:2]:
```suggestion
  "label": "Get started",
```

We follow the Google developer documentation style guide for heading format, which is to use sentence case. 😀

Sentence case is a capitalization style where only the first letter of the first word, proper nouns, and other specific conventions are capitalized, while the rest of the text remains in lowercase.

Here's more info, if helpful! https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings

File: docs/build/get-started/get-started.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_label: Get started
+sidebar_label: Overview
```
Comment (jhaaaa) [inline docs/build/get-started/get-started.md:2]: It looks like some of the content on the Get started page is repeated on this page? What did you have in mind? 😀 Maybe we don't need the Overview page? WDYT?

File: docs/developer-quickstart.md [hunk 4]
```diff
@@ -115,92 +115,3 @@ for await (const message of await xmtp.conversations.streamAllMessages()) {
 }
 //eg. New message from 0xd8dA6BF26964aF9D7eEd9e03E53415D37: gm
 ```
-
```
Comment (jhaaaa) [inline docs/developer-quickstart.md:118]: Just making sure that you want to delete the Troubleshooting and Bots sections? 

If you think they are no longer useful - I trust your judgment! 😀

File: docs/interactive-tutorial.md [hunk 5]
```diff
@@ -6,6 +6,8 @@ description: Learn about the core code you need to build basic messaging with XM
 
 import Tutorial from '/src/components/Tutorial'
 
+<Tutorial />
```
Comment (jhaaaa) [inline docs/interactive-tutorial.md:9]: When I access this page - I see two tutorial sections - do you? 👯

**Review by jhaaaa — APPROVED** [review]
Thank you @fabriguespe for making these improvements to the Doc left nav - and based on feedback from devs! 

I left a few comments and suggestions that I'm hoping you'll consider. 😍

Approving to unblock your merge when you're ready.

File: docs/build/get-started/get-started.md (hunk 3 above)
Comment (fabriguespe) [reply docs/build/get-started/get-started.md:2]: You are right! the get started page was meant to be removed.

File: docs/developer-quickstart.md (hunk 4 above)
Comment (fabriguespe) [reply docs/developer-quickstart.md:118]: Thanks! We still have the Bot section in the overview. The troubleshoot now is removed and is part of the repo issues.

**Pull request merged**